client/server/middleware/logger.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import morgan from 'morgan';
+
+/**
+ * Internal dependencies
+ */
+import { getLogger } from 'server/lib/logger';
+
+const NS_TO_MS = 1e-6;
+
+const logRequest = ( req, res, options ) => {
+	const { requestStart } = options;
+
+	// Equivalent to morgan's `dev` format
+	let fields = {
+		method: morgan.method( req ),
```

Comment:
Do I understand correctly that we no longer use `morgan` as the actual logging middleware for Express.js, but use only its formatting functions?

Reply:
Correct.

We could even drop `morgan` entirely if we want, I'm not 100% sure we want to log exactly the same fields.

Reply:
Most of the `morgan` token functions are fairly trivial, like `method = req => req.method` and the dependency could be eliminated without much effort. Completely optional, it's up to you whether you want to do it in this PR or at all.